scene/main/scene_tree.cpp

```diff
@@ -525,6 +527,32 @@ void SceneTree::init() {
 	MainLoop::init();
 }
 
+void SceneTree::_find_hidden_nodes(Node *p_node, LocalVector<Node *> &r_hidden_nodes) const {
```

Comment:
It does not only find them,but set their visibility to true, so maybe the name should reflect that ?

Reply:
Yup sure, will change. :+1: